tests/Umbraco.Tests.UnitTests/Umbraco.Core/PropertyEditors/SliderPropertyValueEditorTests.cs

```diff
@@ -17,6 +17,9 @@ namespace Umbraco.Cms.Tests.UnitTests.Umbraco.Core.PropertyEditors;
 [TestFixture]
 public class SliderPropertyValueEditorTests
 {
+    public static Guid NewGuid => Guid.NewGuid();
+    public static GuidUdi NewGuidUdi => new GuidUdi(Constants.UdiEntityType.Document, NewGuid);
```

Comment:
Not sure you really need to add these as class level variables.  Given they are each only used in one test, it would be cleaner I think to just create them for the test.